src/main/java/net/neoforged/neoforge/items/wrapper/CombinedInvWrapper.java

```diff
@@ -1,107 +0,0 @@
-/*
- * Copyright (c) Forge Development LLC and contributors
- * SPDX-License-Identifier: LGPL-2.1-only
- */
-
-package net.neoforged.neoforge.items.wrapper;
-
-import net.minecraft.world.item.ItemStack;
-import net.neoforged.neoforge.items.IItemHandlerModifiable;
-
-// combines multiple IItemHandlerModifiable into one interface
-public class CombinedInvWrapper implements IItemHandlerModifiable {
```

Comment:
All this stuff should be kept and deprecated for removal. No need to delete any class in this PR really (except for a few internal ones like the old `VanillaInventoryCodeHooks` - although that one could even be kept working for now).